.github/workflows/build.yml

```diff
@@ -21,10 +21,15 @@ jobs:
           registry: ghcr.io
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
+      - id: docker-metadata
+        uses: docker/metadata-action@v5
+        with:
+          images: ghcr.io/${{ github.repository }}
       - name: ${{ github.ref == 'refs/heads/main' && 'Build and Push Docker image' || 'Build Docker image' }}
         uses: docker/build-push-action@v5
         with:
           push: ${{ github.ref == 'refs/heads/main'}}
-          tags: ghcr.io/${{ github.repository }}:latest
```

Comment:
This fails if there is  an uppercase letter in the org or repo name. This is handled by the metadata action.